[2022-06-03 17:22:12,983] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T17:22:06.148190+00:00 [queued]>
[2022-06-03 17:22:12,995] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T17:22:06.148190+00:00 [queued]>
[2022-06-03 17:22:12,996] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 17:22:12,996] {taskinstance.py:1358} INFO - Starting attempt 1 of 2
[2022-06-03 17:22:12,997] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 17:22:13,012] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): processing_data> on 2022-06-03 17:22:06.148190+00:00
[2022-06-03 17:22:13,018] {standard_task_runner.py:52} INFO - Started process 296 to run task
[2022-06-03 17:22:13,022] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'nyc_bus_breakdown_pipeline', 'processing_data', 'manual__2022-06-03T17:22:06.148190+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/nycbus_etl.py', '--cfg-path', '/tmp/tmpurz8g30u', '--error-file', '/tmp/tmp95wr0_gm']
[2022-06-03 17:22:13,023] {standard_task_runner.py:80} INFO - Job 146: Subtask processing_data
[2022-06-03 17:22:13,087] {task_command.py:370} INFO - Running <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T17:22:06.148190+00:00 [running]> on host 9212bd95e1cf
[2022-06-03 17:22:13,354] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=nyc_bus_breakdown_pipeline
AIRFLOW_CTX_TASK_ID=processing_data
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T17:22:06.148190+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-03T17:22:06.148190+00:00
[2022-06-03 17:22:14,530] {base.py:68} INFO - Using connection ID 's3_conn' for task execution.
[2022-06-03 17:22:14,531] {base_aws.py:210} INFO - Credentials retrieved from extra_config
[2022-06-03 17:22:16,424] {python.py:173} INFO - Done. Returned value was: None
[2022-06-03 17:22:16,442] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=nyc_bus_breakdown_pipeline, task_id=processing_data, execution_date=20220603T172206, start_date=20220603T172212, end_date=20220603T172216
[2022-06-03 17:22:16,489] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-03 17:22:16,534] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
